src/test/java/io/camunda/zeebe/bpmnassert/testengine/EngineClientTest.java

```diff
@@ -98,7 +101,7 @@ public void shouldRequestTopology() {
   @Test
   public void shouldUseBuiltInClient() {
     // given
-    final ZeebeClient zeebeClient = zeebeEngine.createClient();
+    zeebeClient = zeebeEngine.createClient();
```

Comment:
:x: I think it would be better not to overwrite the class field. Instead, we could use a different name here and close it within the test. And then `zeebeClient` is not touched at all, but opened and closed by the general code.

Same for other instances in this class, where the field gets overwritten